corehq/motech/repeaters/signals.py

```diff
@@ -19,6 +20,11 @@ def create_form_repeat_records(sender, xform, **kwargs):
 def create_case_repeat_records(sender, case, **kwargs):
     from corehq.motech.repeaters.models import CaseRepeater
     create_repeat_records(CaseRepeater, case)
+    # assume if there's exactly 1 xform_id that modified the case it's being created
```

Comment:
mentioned in the PR description but this is arguably a hacky check, but am not sure there are better ones. @proteusvacuum had mentioned checking the last action, but I don't think that's actually right since you could (in theory - if there was a bug on mobile?) have a create action after a case was already made?